sdk/nodejs/cmd/run/index.ts

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Enable source map support so we get good stack traces.
+import "source-map-support/register";
```

Comment:
Yes to all! sdk/nodejs/provider/server.ts only (publicly) exports a `main` function, so we can do it there too.

Reply:
Turns out we can't do `sdk/nodejs/provider/server.ts` because that gets imported via the index.ts files.

Reply:
> Turns out we can't do `sdk/nodejs/provider/server.ts` because that gets imported via the index.ts files.

Should we do the import inside `main`?

Reply:
> Should we do the import inside `main`?

Or does that not work because it needs to be at the top of the file?

Reply:
Looking at the source code for it, it patches `Error.prepareStackTrace` and install an uncaught exception handler. I think the reason to want it at the top is so that these apply for things that happen before we get to the register call. For us that's not the case, unless we have a bad bug. 

I think it's fine to do it in main here.